internal/admin/dashboard/static/css/dashboard.css

```
@@ -0,0 +1,408 @@
+    --text-muted: #6b7085;
+    --accent: #4f46e5;
+    --accent-hover: #6366f1;
+    --success: #16a34a;
+    --warning: #d97706;
+    --danger: #dc2626;
+    --chart-grid: #e5e7eb;
+    --chart-text: #6b7085;
+    --chart-tooltip-bg: #ffffff;
+    --chart-tooltip-border: #d8dbe3;
+    --chart-tooltip-text: #1a1d2b;
+    color-scheme: light;
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)